hera_cal/redcal.py

```diff
@@ -832,20 +832,25 @@ def __init__(self, data, sol0, wgts={}, gain=.3, **kwargs):
             **kwargs: keyword arguments of constants (python variables in keys of data that
                 are not to be solved for) which are passed to linsolve.LinProductSolver.
         """
-        linsolve.LinProductSolver.__init__(self, data, sol0, wgts=wgts, **kwargs)
+        linsolve.LinProductSolver.__init__(self, data, sol0, wgts=wgts, build_solver=False, **kwargs)
         self.gain = np.float32(gain)  # float32 to avoid accidentally promoting data to doubles.
 
-    def _get_ans0(self, sol, keys=None):
+    def _get_ans0(self, sol, keys=None, to_update_inplace=None):
```

Comment:
It might be useful just to quickly say what input might be added to the `to_update_inplace` variable. It is a helper function, though, and I don't think the usage is too confusing, but I had to double-check the `solve_iteratively` function to made sure I understood how you were using it

Reply:
Done.